tutorials/XGI in 5 minutes.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>we should try to update the circular layout function so that it's actually a circle, not distorted</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
We can open an issue about that, but do we want that or not? in networkx they don't have it (see example)

<img width="810" alt="Screenshot 2023-07-23 alle 18 42 01" src="https://github.com/xgi-org/xgi/assets/83019028/f2e764ef-ffe2-4fdc-ad03-0d7e17f12d75">

Reply:
Let's see, I opened an issue.